autosklearn/evaluation/__init__.py

```diff
@@ -126,7 +145,7 @@ def __init__(
         resampling_strategy: Union[
             str, BaseCrossValidator, _RepeatedSplits, BaseShuffleSplit
         ],
-        metric: Scorer,
+        metrics: Sequence[Scorer],
```

Comment:
Here we only allow a sequence and no longer a single scorer. I believe that's fine if it is consistent with the rest of the code (where we currently allow both)

Reply:
Which function would that be?

Reply:
They're actually two: `autosklearn.metrics.compute_single_metric` and `autosklearn.metrics._compute_single_scorer`